charts/promitor-agent-resource-discovery/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-version: 0.1.0
+version: 0.2.0
```

Comment:
No need to update this since it's still in preview

Reply:
Ok, reverted.